graphic_templates/bar_chart/js/graphic.js

```diff
@@ -6,17 +6,18 @@ var isMobile = false;
  * Initialize the graphic.
  */
 var onWindowLoaded = function () {
-    if (Modernizr.svg) {
```

Comment:
Whoops this was just me unintentionally editing this. I've fix it. Do I need to do another pull request?